src/WebJobs.Script/Description/Node/Script/globalInitialization.js

```diff
@@ -8,6 +8,9 @@ return function (context, callback) {{
         context.handleUncaughtException(err.stack);
     }});
 
+    // TEMP HACK: workaround for https://github.com/tjanczuk/edge/issues/325
```

Comment:
Please also add a new issue in this repo to remove this hack, referencing the Edge bug